src/hooks/useUSDPrice.ts

```diff
@@ -63,6 +63,7 @@ export function useUSDPrice(currencyAmount?: CurrencyAmount<Currency>): {
     skip: !chain || !isGqlSupportedChain(currency?.chainId),
     pollInterval: PollingInterval.Normal,
     notifyOnNetworkStatusChange: true,
+    fetchPolicy: 'cache-first',
```

Comment:
Clarification: we default to `cache-and-network` in [here](https://github.com/Uniswap/interface/blob/7818426b532181c89201df7ad809ad749596aa45/src/graphql/data/apollo.ts#L55), and this overrides that setting?

Reply:
that's right